parsers/ECO2MIX.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python3
+
+import json
+import math
+import os
+from datetime import datetime, timedelta
+from logging import Logger, getLogger
+
+import arrow
+import numpy as np
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+
+from electricitymap.contrib.config import EXCHANGES_CONFIG, ZONES_CONFIG
```

Comment:
```suggestion
from electricitymap.contrib.config import ZONES_CONFIG
```
Exchange config is unused